int32 map_id
int32 init_step
float64 scale
float64 timestamp